src/components/blocks/CardHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import { UtkwdsCardHeaderBlockAttributes } from 'client';
+
+interface Props {
+  attributes: Partial<UtkwdsCardHeaderBlockAttributes>;
+}
+
+const CardHeader = ({ attributes: { className, content } }: Props) =>
+  content ? (
+    // this also has an attribute of tagName so that you can change the type of container
+    // not sure if we need that or how to implement?
+    // the default was div – so I have just used that for now
+    <div
+      className={`'card-header' ${className || ''}`}
+      dangerouslySetInnerHTML={{ __html: content }}
+    ></div>
```

Comment:
Yes, this bit is not at all obvious, but it turns out to be fairly simple. Relevant part of the docs are here: https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized (though they don't quite explain how to handle this situation).

The thing to do is define a variable—capitalized—that's set to the `tagName` string, and use that variable where you've used `div` above. Something like:

```ts
const CardHeader = ({ attributes: { className, content, tagName: TagName } }: Props) => {
  if (!content || !TagName) return <></>;
  return <TagName /* ... */></TagName>
};
```

Reply:
(There I've actually capitalized `tagName` right in the destructured function-parameters, which I assume would work fine, but you could instead just do a `const TagName = tagName` in the function body.)